microsoft-edge/develop-web-microsoft-edge.md

```diff
@@ -101,9 +101,9 @@ See [Test and automation in Microsoft Edge](test-and-automation/test-and-automat
 
 
 <!-- ====================================================================== -->
-## Web platform
+## Development tips for Microsoft Edge
 
-Considerations for developing websites and products for the web platform include the following:
+There are many tips and tricks to be aware of when building on the web platform, including:
```

Comment:
I think we can simplify `web platform` to `web`, and we somehow need to say that those tips are specific to Edge only. Suggestion:

```suggestion
There are many tips and tricks to be aware of when building web sites or apps that work in Microsoft Edge, including:
```

Reply:
in this top-level doc and the bucket intro:
There are many tips and tricks to be aware of when building websites or web apps that work in Microsoft Edge, including: